client-misk/src/main/kotlin/app/cash/backfila/client/misk/PartitionProvider.kt

```diff
@@ -52,9 +52,9 @@ class UnshardedPartitionProvider(private val transacter: Transacter) : Partition
  * is its own partition. If your entities can move, or if you need to run this backfill slower
  * than one thread per shard, consider using [VitessSingleCursorPartitionProvider] instead.
  */
-class VitessShardedPartitionProvider<E : DbEntity<E>, Pkey : Any>(
+class VitessShardedPartitionProvider<E : DbEntity<E>, Pkey : Any, Param : Any>(
   private val transacter: Transacter,
-  backfill: Backfill<E, Pkey>
+  backfill: Backfill<E, Pkey, Param>
```

Comment:
since you're not using Param in here, could it just be * ? I don't think it really matters though

Reply:
Yeah you are right,

When Partitioning wants access to some parameters (Maybe for Dynamo) then they will need this.